introduction/what-is-sideloading/faq.md

```diff
@@ -14,7 +14,9 @@ Yes. See the page links below.
 
 ## Do I need to buy a developer account?
 
-No! You can use a free one, however, apps will be limited to 7 days, at which point you will have to resign.
+If you want to sideload without revocations and as many apps as you want without computer, yes. Account costs around $99, but, instead of paid code signing services (which starts around $20 **per device**), can be used on **all your devices (current and future)** and devices of your friends or family. So, if you have more than 3 devices or friends, purchasing developer account from Apple suits you best.
```

Comment:
`If you want to sideload without revocations and limitations, yes. Developer accounts costs $99, but instead third-party code signing services (which starts around $20 **per device**), it can be used on **all your devices (current and future)**, as well as devices of your friends or family. If you have more than 3 devices or friends, purchasing a developer account from Apple suits you best.
`